### ISSUE TITLE
func init for Custom Handlers

### ISSUE THREAD

**Issue by anthonychu** [Azure/azure-functions-core-tools#2052]
Parent: #1972 

Add `custom` as a worker runtime stack option.

```
func init --worker-runtime custom
```

If `custom` is selected, add empty `customHandler` section to host.json:

```json
{
  "version": "2.0",
  "logging": {
    "applicationInsights": {
      "samplingSettings": {
        "isEnabled": true,
        "excludedTypes": "Request"
      }
    }
  },
  "extensionBundle": {
    "id": "Microsoft.Azure.Functions.ExtensionBundle",
    "version": "[1.*, 2.0.0)"
  },
  "customHandler": {
    "description": {
      "defaultExecutablePath": "",
      "workingDirectory": "",
      "arguments": []
    },
    "enableForwardingHttpRequest": false
  }
}

```

@pragnagopa Is the schema correct? (forgot if we decided to remove `type`). Thanks.

**Comment by pragnagopa** [comment]
> Is the schema correct? (forgot if we decided to remove type)

Updated description with the right schema. `type` defaults to `http`. We can leave it out of documentation.

**Comment by anthonychu** [comment]
@yojagad @pragnagopa Made a small update to the host.json in the issue. As mentioned in my PR comment #2093 I think `enableForwardingHttpRequest` should be set to the default of `false`.

**Comment by fabiocav** [comment]
Blocked on templates release. Pushing to sprint 81

**Comment by pragnagopa** [comment]
@yojagad  - please verify if the changes do support initializing a docker file `func init LocalFunctionsProject --worker-runtime dotnet --docker` . More details here: https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-linux-custom-image?tabs=bash%2Cportal&pivots=programming-language-csharp

If it requires more work, please create separate issue.

**Comment by fabiocav** [comment]
Work no longer blocked. We should now be able to validate and merge the changes.
